### ISSUE TITLE
how to calculate test data accuracy in keras

### ISSUE THREAD

**Issue by sreevasu** [jocicmarko/ultrasound-nerve-segmentation#56]
hi,  iam taking some of the training data as a test data. How can i  calculate  accuracy of the test data. please any one can you help me.
--

**Comment by knot15** [comment]
you can simply pass your test data in model.fit() to get accuracy each epoch or just calculate the accuracy from the saved weight after the after the training.

**Comment by jocicmarko** [comment]
Depending on how you want to measure accuracy?
